### PR TITLE
Feat/recreate for drivers

### LINKED ISSUE #166: Kubernetes driver rebuild support
ul

### PR DESCRIPTION
**Pull request by 89luca89**
With this PR we add `--recreate` support for custom drivers

Fix #166 
Resolves ENG-1367

### CODE REVIEW

File: pkg/devcontainer/run.go [hunk 1]
```diff
@@ -316,6 +315,17 @@ func runInitializeCommand(
 	return nil
 }
 
+func (r *runner) recreateCustomDriver(ctx context.Context, options UpOptions) (*config.Result, error) {
+	err := r.Driver.StopDevContainer(ctx, r.ID)
+	if err != nil {
+		return nil, errors.Wrap(err, "rebuilding the workspace is currently not supported for this driver")
```
Comment (pascalbreuninger) [inline pkg/devcontainer/run.go:321]: not sure the error message makes sense now 🤔

Comment (89luca89) [reply pkg/devcontainer/run.go:321]: Welp in case we will have other custom drivers where recreate doesn't work :shrug:

Comment (pascalbreuninger) [reply pkg/devcontainer/run.go:321]: but it could also fail for unrelated reasons, because all custom drivers can be stopped

Comment (pascalbreuninger) [reply pkg/devcontainer/run.go:321]: sounds good

Comment (89luca89) [reply pkg/devcontainer/run.go]: done :+1:

**Pull request merged**